README.md

```diff
@@ -47,7 +47,7 @@ sudo yum install -y epel-release && sudo yum install -y git gcc gcc-c++ make sna
 sudo apt-get install gcc g++ make libsnappy-dev autoconf automake libtool which libgtest-dev
 
 # MACOSX
-brew install snappy googletest
+brew install autoconf automake libtool snappy googletest
```

Comment:
do we need to brew install gcc? I find we install gcc in CI for macOS.
https://github.com/KvrocksLabs/kvrocks/blob/unstable/.github/workflows/kvrocks.yaml#L60

Reply:
The original version of GCC in MacOS is `4.2.1`. Although the compilation requirement requires a higher version(gcc >= 4.8), the compilation works around in fact. So, should we recommend the MacOS users upgrade the GCC version?

Reply:
it seems that gcc is clang on macOS platform, it is fine for compilation.